backend/src/modules/idim-webservice/idim-webservice.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export enum RequesterAccountTypeCode {
+    Internal = 'Internal',
+}
+
+export class IDIRUserResponse {
+    @ApiProperty()
+    found: boolean;
+
+    @ApiProperty()
+    userId: string;
+
+    @ApiProperty()
+    displayName: string;
+}
```

Comment:
I wonder if these return values are good enough? Or do we want more information like email etc.?

Reply:
This version just needs a simple yes/no if I recall correctly. Wasn't the initial requirement for the UI just to display a check mark? If so, are we even using displayName or is this looking forward to a future enhancement?